src/test/java/com/google/cloud/pontem/FormatGenericRecordAsSpannerMutationFnTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.pontem;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Type;
+import com.google.common.collect.ImmutableMap;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.generic.GenericRecordBuilder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for {@link FormatGenericRecordAsSpannerMutationFnTest}. */
+@RunWith(JUnit4.class)
+public final class FormatGenericRecordAsSpannerMutationFnTest {
+
+  @Test
+  public void testFormatGenericRecordAsSpannerMutationFn_basic() {
+    String tableName1 = "basicTable1";
+    Schema schema1 =
+        SchemaBuilder.record(tableName1)
+            .namespace("com.google.cloud.pontem")
+            .fields()
+            .name("userName")
+            .type(SchemaBuilder.builder().stringType())
+            .noDefault()
+            .name("userId")
+            .type(SchemaBuilder.builder().longType())
+            .noDefault()
+            .endRecord();
+    GenericRecord genericRecord1 =
+        new GenericRecordBuilder(schema1).set("userName", "John Doe").set("userId", 10L).build();
+    ImmutableMap<String, Type> tableMap1 =
+        ImmutableMap.of("userName", Type.string(), "userId", Type.int64());
+
+    Mutation expectedMutation =
+        Mutation.newInsertOrUpdateBuilder(tableName1)
+            .set("userName")
+            .to((String) "John Doe")
+            .set("userId")
+            .to((long) 10L)
+            .build();
+
+    FormatGenericRecordAsSpannerMutationFn fn =
+        new FormatGenericRecordAsSpannerMutationFn(tableName1, tableMap1);
+    Mutation actualMutation = fn.apply(genericRecord1);
+    assertEquals(expectedMutation, actualMutation);
+  }
+
+  @Test
+  public void testFormatGenericRecordAsSpannerMutationFn_basic1() {
```

Comment:
Is this numbered approach the regular naming convention? I'd go for something a bit more descriptive ditto for the following test classes